aspnetcore/fundamentals/error-handling.md

```diff
@@ -61,6 +61,36 @@ public IActionResult Error()
 
 Don't decorate the error handler action method with HTTP method attributes, such as `HttpGet`. Explicit verbs prevent some requests from reaching the method. Allow anonymous access to the method so that unauthenticated users are able to receive the error view.
 
+## Access the exception
+
+If you need to access the exception or the original request path in a controller or page, use <xref:Microsoft.AspNetCore.Diagnostics.IExceptionHandlerPathFeature>:
+
+* The path is available from the <xref:Microsoft.AspNetCore.Diagnostics.IExceptionHandlerPathFeature.Path> property.
+* Read the <xref:System.Exception?displayProperty=fullName> from the inherited [IExceptionHandlerFeature.Error](xref:Microsoft.AspNetCore.Diagnostics.IExceptionHandlerFeature.Error) property.
+
+```csharp
+// using Microsoft.AspNetCore.Diagnostics;
+
+var exceptionHandlerPathFeature = 
+    HttpContext.Features.Get<IExceptionHandlerPathFeature>();
+var path = exceptionHandlerPathFeature?.Path;
+var error = exceptionHandlerPathFeature?.Error;
+```
+
+> [!WARNING]
+> Do **not** serve sensitive error information from <xref:Microsoft.AspNetCore.Diagnostics.IExceptionHandlerFeature> or <xref:Microsoft.AspNetCore.Diagnostics.IExceptionHandlerPathFeature> to clients. Serving sensitive information from errors can compromise the app's security.
+
+## Configure custom exception handling code
+
+An alternative to serving an endpoint for errors with a [custom exception handling page](#configure-a-custom-exception-handling-page) is to provide a lambda to <xref:Microsoft.AspNetCore.Builder.ExceptionHandlerExtensions.UseExceptionHandler*>. This approach permits the app to work directly with the error before returning a response.
+
+The sample app demonstrates the approach in `Startup.Configure`. Trigger an exception with the **Throw Exception** link on the Index page. The following lambda runs:
```

Comment:
The sample app demonstrates this approach in `Startup.Configure`. Trigger an exception with the **Throw Exception** link on the Index page. The following lambda runs: